ipywidgets/widgets/widget_int.py

```diff
@@ -150,6 +150,7 @@ class IntSlider(_BoundedInt):
         default_value='horizontal', help="Vertical or horizontal.").tag(sync=True)
     _range = Bool(False, help="Display a range selector").tag(sync=True)
     readout = Bool(True, help="Display the current value of the slider next to it.").tag(sync=True)
+    readout_format = Unicode('d', help="Format for the readout").tag(sync=True)
```

Comment:
Can we put in a short link to the readout format in the IntSlider docstring? Perhaps just `The `readout_format` format specification is at https://github.com/mbostock/d3/wiki/Formatting#numbers.`